New version: FlightGear.FlightGear version 2024.1.3

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #314133

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

> [!NOTE]
>
> This package is a nested installer. Around 1735MB external files needs to be downloaded inside the installer.

<img width="1666" height="628" alt="Image" src="https://github.com/user-attachments/assets/0415f8e5-2ac8-4293-a49e-fd112df2e0c0" />
<img width="1720" height="1121" alt="Image" src="https://github.com/user-attachments/assets/e01e6b62-3f09-4077-9b18-0f601d3f0ce2" />

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/314167)

Dragon1573 committed:
New version: FlightGear.FlightGear version 2024.1.3

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

AsciiWolf commented on manifests/f/FlightGear/FlightGear/2024.1.3/FlightGear.FlightGear.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+  DefaultInstallLocation: '%ProgramFiles%\FlightGear 2024.1'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirrors.ibiblio.org/flightgear/ftp/release-2024.1/flightgear-2024.1.3-windows-amd64.exe
```
```suggestion
  InstallerUrl: https://sourceforge.net/projects/flightgear/files/release-2024.1/flightgear-2024.1.3-windows-amd64.exe
```

AsciiWolf replied:
(Just like the older winget manifests.)

Dragon1573 committed:
Apply suggestions from code review

- Switch installer URL to SourceForge
- Keep consistency with previous manifests

Co-authored-by: AsciiWolf <mail@asciiwolf.com>
